HolyshiftService/PasswordHashService.cs

```diff
@@ -0,0 +1,30 @@
+using Konscious.Security.Cryptography;
+using System;
+using System.Text;
+
+public class PasswordHashService
+{
+    public string GenerateRandomSalt(int size)
+    {
+        byte[] salt = new byte[size];
+        using (var rng = System.Security.Cryptography.RandomNumberGenerator.Create())
+            rng.GetBytes(salt);
+
+        return Convert.ToBase64String(salt);
+    }
+
+    public async Task<string> HashPassword(string password, string saltBase64)
+    {
+        byte[] salt = Convert.FromBase64String(saltBase64);
+
+        using var hasher = new Argon2id(Encoding.UTF8.GetBytes(password));
```

Comment:
Add a comment above this line stating that this is one of the recommended settings by OWASP and link to the document
https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html#argon2id